cluster/manifests/roles/scalyr-key-rotation-rbac.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Why is this a ClusterRole? It's only used in one namespace, so it should be a Role.